data/mods/chatbats/moves.ts

```diff
@@ -872,4 +872,206 @@ export const Moves: import('../../../sim/dex-moves').ModdedMoveDataTable = {
 		desc: "Changes the target's type to Steel.",
 		shortDesc: "Changes the target's type to Steel.",
 	},
+	triplekick: {
+		inherit: true,
+		basePowerCallback(pokemon, target, move) {
+			return 20 * move.hit;
+		},
+	},
+	freezingglare: {
+		inherit: true,
+		onHit(target, source, move) {
+			if (this.randomChance(3, 10) && !target.hasType('Ice')) {
+				target.setStatus('frostbite', source, move);
+			}
+		},
+		secondary: null,
+		desc: "30% chance to inflict Frostbite.",
+		shortDesc: "30% chance to inflict Frostbite.",
+	},
+	zippyzap: {
+		inherit: true,
+		category: "Special",
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Extreme Speed', target);
+			this.add('-anim', source, 'Thunder', target);
+		},
+		secondary: null,
+		desc: "",
```

Comment:
```suggestion
		desc: "Nearly always goes first.",
```
You didn't adjust priority so this is still true